select_test.go

```diff
@@ -46,11 +47,13 @@ func TestGetSelectedRange(t *testing.T) {
 
 func TestGetTextRange(t *testing.T) {
 	// Prepare the text grid for the tests
-	grid := &widget.TextGrid{
-		Rows: []widget.TextGridRow{
-			{Cells: []widget.TextGridCell{{Rune: 'A'}, {Rune: 'B'}, {Rune: 'C'}}},
-			{Cells: []widget.TextGridCell{{Rune: 'D'}, {Rune: 'E'}, {Rune: 'F'}}},
-			{Cells: []widget.TextGridCell{{Rune: 'G'}, {Rune: 'H'}, {Rune: 'I'}}},
+	grid := widget2.TermGrid{
```

Comment:
You don't need to init the inner type - it is there anyway. Try:

```go
grid := widget2.NewTermGrid()
grid.Rows = []widget.TextGridRow{ ... }
```

Reply:
fixed thank you :D